profiles/logging.py

```diff
@@ -5,6 +5,9 @@
 
 from pythonjsonlogger import jsonlogger
 
+sql_engine_logger = logging.getLogger('sqlalchemy.engine')
+app_logger = logging.getLogger(__name__)
```

Comment:
I think this is not equivalent to the original `logging.getLogger()`? That gets the root Logger object while this one gets the `profiles.logging` Logger, whose configuration won't cascade into other Logger objects

Reply:
I have reverted the code and, after resolving other configuration issues, this appears to be working as expected.